samples/CosmosDB/audit-if-not-exists-cosmosdb-ip-range-filter/azurepolicy.json

```diff
@@ -0,0 +1,29 @@
+{
+  "properties": {
+    "mode": "all",
+    "displayName": "Audit IP Range Filter for CosmosDB accounts",
+    "description": "This policy audits that the IP Range Filter attribute is configured for CosmosDB accounts",
+    "policyRule": {
+      "if": {
+        "field": "type",
+        "equals": "Microsoft.DocumentDB/databaseAccounts"
+      },
+      "then": {
+        "effect": "AuditIfNotExists",
+        "details": {
+          "type": "Microsoft.DocumentDB/databaseAccounts",
+          "existenceCondition": {
+            "allOf": [
+              {
+                "field": "Microsoft.DocumentDB/databaseAccounts/ipRangeFilter",
+                "notEquals": ""
+              }
+            ]
+          }
+        }
+      }
+    },
```

Comment:
This should be done as a regular "Audit" policy.  "AuditIfNotExists" is only needed when targeting two different resourceTypes.

Reply:
Hey @camillemarie, I dropped the "AuditIfNotExists" policy and replaced with an "Audit" policy.